lib/command.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+var mson2json = require('./mson2json');
+var fs        = require('fs');
+var path      = require('path')
+
+var M2jCommand = (function() {
```

Comment:
instead of assigning to this var, you can just do:

```JavaScript
module.exports = (function () {
  // all your logic
  return M2jCommand;
})();
```